arch/arm64/configs/xiaomi-tissot.config

```diff
@@ -1,9 +1,11 @@
 CONFIG_BACKLIGHT_QCOM_WLED=y
-CONFIG_BATTERY_QCOM_FG=m
+CONFIG_BATTERY_QCOM_FG=y
+CONFIG_CHARGER_QCOM_SMBCHG=y
 CONFIG_BMI160_I2C=m
-CONFIG_DRM_PANEL_FT8716_FHD=y
-CONFIG_DRM_PANEL_ILI7807_FHD=y
-CONFIG_DRM_PANEL_OTM1911_FHD=y
+CONFIG_AK8975=m
+CONFIG_DRM_PANEL_MDSS_FT8716_FHD=y
```

Comment:
They will be installed by modules automatically because they depend on drm and msm_drm which are set as modules in defconfig

Reply:
msm_drm was switched to module due to tissot as it caused kernel panic. This problem has been fixed with the latest kernel releases

Reply:
I really don't care. No problem if @alikates wants it too

Reply:
> If someone decides to switch drm to build-in, it will be less painful, since it will not be necessary to enable build-in for panels again

got it, what about the light sensor?

Reply:
> > If someone decides to switch drm to build-in, it will be less painful, since it will not be necessary to enable build-in for panels again
> 
> got it, what about the light sensor?

I think we need to make some adjustments to the driver, because it still does not work correctly. We will also need to add a new compatible to the driver, because ltr579 still has some differences. I suggest not touching ltr579 in dts for now. We'll fix this in the next release